examples/asrc_demo/src/rate_server.c

```diff
@@ -172,50 +186,33 @@ static float_s32_t determine_avg_I2S_rate_from_driver()
     return result;
 }
 
-void calc_avg_i2s_send_buffer_level(int current_level, bool reset)
+static inline sw_pll_q24_t get_Kp_for_i2s_buffer_control(int32_t nominal_i2s_rate)
 {
-    static int64_t error_accum = 0;
-    static int32_t count = 0;
-
-    if(reset == true)
+    // The Kp constants are generated using the simulation framework, largely through trial and error, to get values using which
```

Comment:
perhaps say "empirically" - it's a bit less scary than saying "trial and error!"